https://bugs.gentoo.org/580702

fix build with newer glibc where sys/sysmacros.h isn't implicitly included
by sys/types.h

--- a/src/DNotify.c++
+++ b/src/DNotify.c++
@@ -32,6 +32,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <sys/sysmacros.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <libgen.h>
